R/tm_g_waterfall.R

```diff
@@ -265,14 +265,15 @@ ui_g_waterfall <- function(id, ...) {
         value = a$gap_point_val
       )
     ),
-    forms = get_rcode_ui(ns("rcode")),
+    forms = teal.widgets::verbatim_popup_ui(ns("rcode"), "Show R code"),
     pre_output = a$pre_output,
     post_output = a$post_output
   )
 }
 
 srv_g_waterfall <- function(id,
-                            datasets,
+                            data,
```

Comment:
In the roxygen example set the following 3 options and you get this:

![image](https://user-images.githubusercontent.com/15201933/187869671-6d8ff5a8-c66b-45a7-a223-6b313d0db5e8.png)

Reply:
same on main - i made an issue

https://github.com/insightsengineering/teal.osprey/issues/176